core/web/resolver/p2p.go

```diff
@@ -0,0 +1,96 @@
+package resolver
+
+import "github.com/smartcontractkit/chainlink/core/services/keystore/keys/p2pkey"
+
+type P2PKey struct {
+	id        string
```

Comment:
Any reason to not use use the key here as a field?

`key p2pkey.KeyV2`

This is preferable because then you can do any type conversions necessary in the getter

Reply:
No good reason. You're absolutely right!